backend/database/database.go

```diff
@@ -1345,7 +1347,7 @@ type AreaByRating struct {
 func (d *Database) PopularVendor() ([]AreaByRating, error) {
 	const command = `
 					SELECT Vendor.Name as 'BusinessName' , AreaName as 'Location',
-					count(distinct(StarRating)) as "TotalRatings"
+					count(distinct(StarRating)) as "TotalRating"
```

Comment:
You have a `TotalRatings` field in `AreaByRating`, so you should change the above column name to match that.